### ISSUE TITLE
Object returned from function should be considered local?

### ISSUE THREAD

**Issue by danatemple** [sloops77/eslint-plugin-better-mutation#26]
This code should not throw an error I think, as the keys array is locally generated:

```
    Object.keys(myObject).sort(mySortFn)
```

**Comment by sloops77** [comment]
Addressed in #70
